### PR TITLE
Add Dockerfile

### PR DESCRIPTION
**Pull request by cmsj**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -4,5 +4,8 @@ WORKDIR /usr/src/tarssh
 COPY . .
 
 RUN cargo install --path .
+RUN mkdir /var/empty
+RUN chown nobody:nogroup /var/empty
```
Comment (Freaky) [inline Dockerfile:8]: Should be `root:wheel` - `nobody` should never own files.

File: Dockerfile [hunk 2]
```diff
@@ -4,5 +4,8 @@ WORKDIR /usr/src/tarssh
...
+RUN chmod u=rx,g=rx,o-rwx /var/empty
```
Comment (Freaky) [inline Dockerfile:9]: `rx` should do, 0555?

Seems odd it's not included in the environment by default.

**Comment by Freaky** [comment]
I should probably learn more about Docker before merging this.  Seems a bit weird for an environment not to include something as simple as /var/empty :/

**Comment by Freaky** [comment]
Merged, but replaced with a fancier one with a separate build step, use of `ENTRYPOINT`, `EXPOSE`, etc.  Guess I should see about some official image now or something?

**Comment by cmsj** [comment]
Docker Hub can do automated builds for free :)

**Pull request merged**